### PR TITLE
feat: 🎸 use react-native-jose

### PR DESCRIPTION
**Pull request by raduachim**
- use [native implementation of methods for JOSE](https://github.com/egendata/react-native-jose)

- [x] `sign`
- [x] `verify`
- [x] `decode`
- [x] `remove jwt-lite`
- [x] `iOS`
- [x] `Android`

How to test locally:

- Android: 
    - `npm i`
    - `npm start`
    - `npm run android` (atleast needed first time since it runs Jetifier to migrate libraries to AndroidX; after that you can run it from Android Studio)
    - if you make changes inside the react-native-jose, sometimes, due to some caching I assume, the app crashes on start (so go to `MainApplication.java` file and comment out `new JosePackage()` and build the app, then uncomment it back and build again and it should clear the cache and build it)
    - if you want to run it on an actual device you need to run `adb reverse tcp:8081 tcp:8081` so that the phone can reach the Metro bundler

- iOS:
    - `npm i`
    - `cd ios && pod install && cd ..`
    - open the workspace in XCode and then you can choose to run it on a simulator or directly on an iPhone

### CODE REVIEW

File: lib/services/tokens.js [hunk 1]
```diff
@@ -3,12 +3,19 @@ import { getAccount, getAccountKeys } from './storage'
 import Config from 'react-native-config'
 import { schemas } from '@egendata/messaging'
 
-const nowSeconds = () => Math.round(Date.now() / 1000)
+const nowSeconds = () => Math.floor(Date.now() / 1000)
+
+function pem2der(key) {
```
Comment (lessp) [inline lib/services/tokens.js]: 😍

**Comment by iteadam** [comment]
I am unable to login to cv-test (using operator-test) with this version of the App. Everything seems to work, except the CV backend never returns anything other than 404 to the front end.

It could be that CV backend is rejecting the consent sent by the app, or something else. I'd need to run CV locally to debug.

To make matters worse the version of the App in the app store is broken as well

**Comment by raduachim** [comment]
> I am unable to login to cv-test (using operator-test) with this version of the App. Everything seems to work, except the CV backend never returns anything other than 404 to the front end.
> 
> It could be that CV backend is rejecting the consent sent by the app, or something else. I'd need to run CV locally to debug.
> 
> To make matters worse the version of the App in the app store is broken as well

You must be doing something wrong.

I've tested it running operator and cv locally and it works

I've tested with operator-ci (`OPERATOR_URL=https://operator-ci.dev.services.jtech.se`) and cv-ci and it works.

I've tested with operator-test - this one seems to not have been released for a while so the URL for it that I tested with is `OPERATOR_URL=https://operator-test.dev.services.jtech.se/api` and it worked

I've released now the operator-test to latest version and tested it with `OPERATOR_URL=https://operator-test.dev.services.jtech.se` and it works

**Comment by iteadam** [comment]
> > I am unable to login to cv-test (using operator-test) with this version of the App. Everything seems to work, except the CV backend never returns anything other than 404 to the front end.
> > It could be that CV backend is rejecting the consent sent by the app, or something else. I'd need to run CV locally to debug.
> > To make matters worse the version of the App in the app store is broken as well
> 
> You must be doing something wrong.
> 
> I've tested it running operator and cv locally and it works
> 
> I've tested with operator-ci (`OPERATOR_URL=https://operator-ci.dev.services.jtech.se`) and cv-ci and it works.
> 
> I've tested with operator-test - this one seems to not have been released for a while so the URL for it that I tested with is `OPERATOR_URL=https://operator-test.dev.services.jtech.se/api` and it worked
> 
> I've released now the operator-test to latest version and tested it with `OPERATOR_URL=https://operator-test.dev.services.jtech.se` and it works

`/api` that was it... :man_facepalming:

**Pull request merged**